<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>435</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>609</width>
    <height>343</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ExCiton 0.2</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>2.ico</normaloff>2.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>601</width>
      <height>315</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>191</width>
            <height>133</height>
           </size>
          </property>
          <property name="title">
           <string>Характеристики фононів</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>17</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Енергія фонона, еВ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_fonona"/>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>133</height>
           </size>
          </property>
          <property name="title">
           <string>Діелектрична проникливість матеріалу</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>191</width>
             <height>111</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>13</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Статична</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="eps_0"/>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>17</width>
                <height>13</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Високочастотна</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="eps_nesk"/>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string/>
       </property>
       <widget class="QCheckBox" name="chRandomAngle">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>210</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Рандомний кут</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="n_iter">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>50</y>
          <width>265</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="obl_localizacii">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>96</y>
          <width>265</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="temperatura">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>281</y>
          <width>265</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="kut">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>235</y>
          <width>265</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="napr_polia">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>189</y>
          <width>265</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="koef_elektr_sporidn">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>142</y>
          <width>265</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>261</y>
          <width>87</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Температура</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>169</y>
          <width>126</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Напруженість поля</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>122</y>
          <width>259</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Коефіцієнт електронного споріднення</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>215</y>
          <width>28</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Кут </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>76</y>
          <width>129</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Область локалізації</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>119</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Кількість ітерацій</string>
        </property>
       </widget>
       <zorder>n_iter</zorder>
       <zorder>obl_localizacii</zorder>
       <zorder>temperatura</zorder>
       <zorder>kut</zorder>
       <zorder>napr_polia</zorder>
       <zorder>koef_elektr_sporidn</zorder>
       <zorder>label_10</zorder>
       <zorder>label_8</zorder>
       <zorder>label_4</zorder>
       <zorder>label_9</zorder>
       <zorder>label_7</zorder>
       <zorder>label_6</zorder>
       <zorder>chRandomAngle</zorder>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="showGraph">
         <property name="text">
          <string>Графік</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showTable">
         <property name="text">
          <string>Таблиця</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="b_modeling">
       <property name="text">
        <string>Змоделювати</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>197</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Метод розрахунку інтегралу</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="CheckIntegral"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>197</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="2" column="0">
      <widget class="QComboBox" name="CheckLow"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Закон росподілу</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Врахувати наявність точкових дефектів</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>112</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Вибір кристалу</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="CheckCristal"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>270</y>
      <width>266</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Кількість ЕЕ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>290</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="ev_open"/>
    <addaction name="ev_save"/>
    <addaction name="ev_saveAs"/>
    <addaction name="separator"/>
    <addaction name="ev_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="ev_manual"/>
    <addaction name="separator"/>
    <addaction name="ev_about"/>
    <addaction name="upgrade"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="ev_open">
   <property name="text">
    <string>Відкрити конфігураційний файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="ev_save">
   <property name="text">
    <string>Зберегти</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="ev_saveAs">
   <property name="text">
    <string>Зберегти як</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="ev_exit">
   <property name="text">
    <string>Вихід</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="ev_manual">
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="ev_about">
   <property name="text">
    <string>Про програму</string>
   </property>
  </action>
  <action name="upgrade">
   <property name="text">
    <string>Оновити</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
